local/run_flink_app.py

```diff
@@ -0,0 +1,11 @@
+from environment import set_aws_credentials, load_environment_variables
+
+
+load_environment_variables()
+set_aws_credentials()
+
+
+if __name__ == "__main__":
+    from flink_app import main
+
+    main()
```

Comment:
I have removed the local scrips for reading and sending records as they are already included in `event_generation/`. Would it make sense to move this local folder for running the app within `flink_app/`?

Reply:
Sounds good in my opinion